homeassistant/helpers/condition.py

```diff
@@ -166,6 +170,17 @@ def if_or_condition(
 or_from_config = _threaded_factory(async_or_from_config)
```

Comment:
This is now broken because `async_or_from_config` is a coroutine function. Is this used anywhere?

Reply:
Removed.